app/helpers/application_helper.rb

```diff
@@ -3,31 +3,68 @@ module ApplicationHelper
   ANY_DAAC_KEYWORD = 'DAAC: ANY'
   ANY_CAMPAIGN_KEYWORD = 'CAMPAIGN: ANY'
   SELECT_DAAC = 'Select DAAC'
-  #providers are specified to identify only the records within EOSDIS
-  PROVIDERS = ['NSIDCV0',
-              'ORNL_DAAC',
-              'LARC_ASDC',
-              'LARC',
-              'LAADS',
-              'GES_DISC',
-              'GHRC',
-              'SEDAC',
-              'ASF',
-              'LPDAAC_ECS',
-              'LANCEMODIS',
-              'NSIDC_ECS',
-              'OB_DAAC',
-              'CDDIS',
-              'LANCEAMSR2',
-              'PODAAC']
-
-
-  def provider_select_list
-    daac_list(ANY_DAAC_KEYWORD)
+
+  #providers are specified to identify only the records within EOSDIS which ARC team curates
+  ARC_PROVIDERS = ['NSIDCV0',
+                   'ORNL_DAAC',
+                   'LARC_ASDC',
+                   'LARC',
+                   'LAADS',
+                   'GES_DISC',
+                   'GHRC',
+                   'SEDAC',
+                   'ASF',
+                   'LPDAAC_ECS',
+                   'LANCEMODIS',
+                   'NSIDC_ECS',
+                   'OB_DAAC',
+                   'CDDIS',
+                   'LANCEAMSR2',
+                   'PODAAC',
+                   'ARCTEST']
+
+  #providers are specified to identify only the records within EOSDIS which MDQ team curates
+  MDQ_PROVIDERS = ['SCIOPS',
+                   'NOAA_NCEI',
+                   'JAXA',
+                   'ISRO',
+                   'AU_AADC',
+                   'ESA',
+                   'EUMETSA',
+                   'MDQTEST']
+
+  def application_mode
```

Comment:
Comments on how this works and all possible modes

Reply:
the only thing that comes to mind is maybe using a state machine in case we change the logic in the future, but I don't think that is an exact fit either. your implementation is fine.

Reply:
so all daac_curators go into arc_mode? this isn't explained in the comment

Reply:
Added a little clearer comments on how the application_mode is determined.

Reply:
This conversation returns me to "I dislike how much we use curator in this application".  

mdq_curator? returns true when either a user has ARC Curator like privileges in non-EOSDIS providers or when their associated daac is in the non-EOSDIS provider list.  In this case, both "Reviewers" and "Reviewees" are MDQ Curators.

Reply:
I think the case on the ground is that we will have no DAAC level MDQ users at the moment, but I think we will need to come up with a clear way of differentiating the two levels of MDQ curators when we do.

Reply:
well the goal here is to determine the mode... i.e., is it mdq_mode or arc_mode.   not necessarily whether they are a curator or not.

Reply:
so it might be mixing/confusing things by saying the "current_user" is a mdq_curator

Reply:
Wonder if this should be renamed now, mdq_user?

Reply:
I think renamed this actually makes sense and goes more with how it is being used.  Agree?